packages/teleport/src/Discover/SelectResource/DatabaseResource.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { Box, Flex, Text } from 'design';
+
+import { Cloud } from 'design/Icon';
+
+import SlideTabs from 'design/SlideTabs';
+
+import {
+  ResourceLocation,
+  ResourceType,
+} from 'teleport/Discover/resource-lists';
+import { ActionButtons } from 'teleport/Discover/Shared';
+
+import { PermissionsErrorMessage } from './PermissionsErrorMessage';
+
+interface DatabaseResourceProps {
+  disabled: boolean;
+  onProceed: () => void;
+}
+
+export function DatabaseResource(props: DatabaseResourceProps) {
+  // As we're focusing on the server flow uncomment this when we start
+  // implementing the database support.
+  // let content = (
+  //   <SelectDBDeploymentType
+  //     selectedType={selectedType}
+  //     setSelectedType={setSelectedType}
+  //     resourceTypes={resourceTypes}
+  //   />
+  // );
+  let content;
+  if (props.disabled) {
+    content = (
+      <PermissionsErrorMessage
+        action="add new Databases"
+        productName="Database Access"
+      />
+    );
+  }
+
+  return (
+    <>
+      {content}
+
+      <ActionButtons
+        onProceed={() => props.onProceed()}
+        disableProceed={props.disabled}
+      />
+    </>
+  );
+}
+
+interface SelectDBDeploymentTypeProps {
+  selectedType: string;
+  setSelectedType: (string) => void;
+  resourceTypes: ResourceType[];
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
maybe put a TODO to remove this in the future

Reply:
There's a corresponding TODO here - https://github.com/gravitational/webapps/pull/1126/files#diff-83b877841f310708c8f9afd4dfe413f40b709e1fe4de6eb42d8a2e643dcdb82dR24-R32